index.html

```diff
@@ -12,8 +12,7 @@
   </head>
   <body>
       <header>
-        <!--  The <nav> tag defines a set of navigation links.
-Notice that NOT all links of a document should be inside a <nav> element. The <nav> element is intended only for major block of navigation links.      -->
+        <!--  Begin HEADER     -->
         <nav class="navbar">
           <a class="logo-osvaldo" href="https://github.com/OsvaldoBC/">Osvaldo</a>
           <a class="box-menu" href="#Menu"><img src="./img/Union.png" alt="Menu"></a>
```

Comment:
- [OPTIONAL] Please make sure that navigation items do not appear in the mobile version 💯
 
![5](https://user-images.githubusercontent.com/45231276/187037480-cc559902-aa2c-4bae-ba0f-b33aff07dab6.PNG)